### PR TITLE
[@mantine/core] Timeline: Add `radius` prop

### PR DESCRIPTION
**Pull request by jerebtw**

### CODE REVIEW

File: docs/src/docs/core/Timeline.mdx [hunk 1]
```diff
@@ -34,3 +34,7 @@ Control timeline appearance with the following props:
 ## Bullet as React node
 
 <Demo data={TimelineDemos.bullet} />
+
+## Radius
+
+<Demo data={TimelineDemos.radius} />
```
Comment (rtivital) [inline docs/src/docs/core/Timeline.mdx:40]: This demo should be a part of configurator

Comment (jerebtw) [reply docs/src/docs/core/Timeline.mdx]: Okay I removed it

File: src/mantine-core/src/components/Timeline/demos/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export { usage } from './usage';
 export { configurator } from './configurator';
 export { bullet } from './bullet';
+export { radius } from './radius';
```
Comment (rtivital) [inline src/mantine-core/src/components/Timeline/demos/index.ts:4]: Remove radius demo

**Comment by rtivital** [comment]
Thanks!

**Pull request merged**